pages/aiheet.md

```diff
@@ -4,36 +4,36 @@ You can either come up with your own topic or pick one from the ones below. The
 
 ## Graphs and pathfinding
 
-* How to find the fastest way possible out of a labyrinth. The labyrinth could be ascii-art or a picture.
+* How to find the fastest way possible out of a labyrinth. The labyrinth could be ASCII-art or a picture.
 
-* How to find the fastest/shortest path between two nodes in a grapth. The nodes could be for isntance street addresses, bus stops or coordinates. See this article if intrested: http://theory.stanford.edu/~amitp/GameProgramming/AStarComparison.html
+* How to find the fastest/shortest path between two nodes in a grapth. The nodes could be for isntance street addresses, bus stops or coordinates. See this article if interested: http://theory.stanford.edu/~amitp/GameProgramming/AStarComparison.html
 * Note: Just A-star is not a sufficient topic. But you could for instance compare different path finding algorithms.
 
 * Maps for pathfinding can be found at [Moving AI Lab](http://www.movingai.com/benchmarks/) and the [downloads site](http://kartat.kapsi.fi/) for the National Land Survey of Finland.
 * Google maps and open street maps also provide usable data.
 
 ## Data compression
 
-* How to compress a given file as efficiently as possible? A desired outcome would be at least a 40-60% reduction in size. And the file has to be inflatable back to the original. How well can your implementation fare compared to existing solutions.
+* How to compress a given file as efficiently as possible? A desired outcome would be at least a 40-60 % reduction in size. And the file has to be inflatable back to the original. How well can your implementation fare compared to existing solutions?
     * Huffman
-    * Lempel Ziv
+    * Lempel–Ziv–Welch
```

Comment:
```suggestion
    * Lempel Ziv
```

The topic doesn't have to be specifically the LZW version. It could also be some other Lempel ZIv variant.